contracts/libraries/SafeCast.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.6.0 <0.8.0;
+
+/**
+ * @dev Wrappers over Solidity's uintXX/intXX casting operators with added overflow
+ * checks.
+ *
+ * Logic was copied and modified from here: https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/SafeCast.sol
+ */
+library SafeCast {
+    function toUint96(uint256 value) internal pure returns (uint96) {
+        require(value < 2**95, "SafeCast: value doesn't fit in 96 bits");
```

Comment:
```suggestion
        require(value < 2**96, "SafeCast: value doesn't fit in 96 bits");
```

Reply:
Ups, I must have peeked at toInt , instead of toUint. sorry